.github/workflows/build-artifacts.yml

```diff
@@ -0,0 +1,151 @@
+name: build-artifacts.yml
+permissions:
+  contents: read
+  id-token: write
+  attestations: write
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'dev/*'
```

Comment:
I think we don't want to make a build for all pushes to any branch. Just merges to main or mergestto hotfix